packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -172,6 +176,19 @@ const useAddProviderFormState = (
   };
 };
 
+// if InsecureSkipVerify field is checked, avoid validation of the caCert
+// fields by cleaning it since the field is hidden.
+// Re-validate only when InsecureSkipVerify is unchecked
+const handleCaCertForInsecure = (
+  caCertField: IValidatedFormField<String> | null,
+  caCertInvisibleDummyField: IValidatedFormField<String> | null,
+  isInsecureSkipVerifyChecked: boolean
+) => {
+  isInsecureSkipVerifyChecked ?
+    (caCertInvisibleDummyField.setValue(caCertField.value), caCertField.setValue('')) :
```

Comment:
`caCertInvisibleDummyField` and `caCertField` can be `null` , according to the type

```suggestion
    if (!caCertField || !caCertInvisibleDummyField) {
      return;
    }
    (caCertInvisibleDummyField.setValue(caCertField.value), caCertField.setValue('')) :
```